dotcom-rendering/src/web/components/FixedSmallSlowIII.tsx

```diff
@@ -0,0 +1,68 @@
+import { ArticleDesign } from '@guardian/libs';
+import { Card } from './Card/Card';
+import { LI } from './Card/components/LI';
+import { UL } from './Card/components/UL';
+
+type Props = {
+	trails: TrailType[];
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+};
+
+export const FixedSmallSlowIII = ({
+	trails,
+	containerPalette,
+	showAge,
+}: Props) => {
+	const smallTrails = trails.slice(0, 3);
+
+	return (
+		<UL direction="row">
+			{smallTrails.map((trail, index) => {
+				return (
+					<LI
+						padSides={true}
+						showDivider={index > 0}
+						padBottomOnMobile={true}
+						percentage={index === 0 ? '50%' : '25%'}
```

Comment:
having these properties decided inline is a bit awkward, but the obvious alternative (having two distinct component calls which hard-code all of their relevant values) is a bit awkward too, as it makes it difficult to see what the cards in this container have in common. Given that the decision-rule here is always based on the same criterion ("is it the first card or not?") having the logic inline doesn't seem to bad to me here, but interested to hear people's thoughts!

Reply:
Thanks Ash! I like your suggestion, do we know roughly how many options there would need to be? My only concern is that the system would need to be designed well to avoid recreating the existing complexity, adding multiple override props, etc. Personally I don't know whether/how that would be possible at the moment as I've not looked into the existing combinations of props in enough detail. But were you saying that Frontend already has an abstraction for this? I'd be interested to check that out if you know where to look?

Reply:
Thats right :) Not saying that you should be doing that in this PR, it would be quite a big task to do this.

Reply:
Okay yep that sounds good, thanks!